conf/broker.conf

```diff
@@ -479,6 +479,9 @@ webSocketSessionIdleTimeoutMillis=300000
 # Enable topic level metrics
 exposeTopicLevelMetricsInPrometheus=true
 
+# Enable consumer level metrics
+# exposeConsumerLevelMetricsInPrometheus=true
```

Comment:
for default value, better also change it to false here.

Reply:
ok